### PR TITLE
Bump version to 2.8.0

### PR DESCRIPTION
**Pull request by tilne**
Signed-off-by: Tim Lane <tilne@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,60 @@
...
+* Enable configuration of the automatic backup features of FSx file systems.
+* Install  PMIx v3.1.5 and provide slurm support for it on all supported operating systems except for
+  CentOS 6.
+* Install glibc-static, which is required to support certain options for the Intel MPI compiler.
+
+**CHANGES**
+
+* Migrate S3 URLs to Virtual Hosted Style Access.
+* Remove hardcoded account IDs from util/generate-ami-list.py.
+* Avoid validation for ParallelCluster required additional IAM policies.
```
Comment (mauri-melato) [inline CHANGELOG.rst:26]: It doesn't sound good, is this a bug fix? what's the purpose of the change

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,60 @@
...
+* Removed unused dependency on supervisor from the Batch Dockerfile.
+* Move all LogGroup definitions in the CloudFormation templates into the CloudWatch substack.
+* Drop dependency on the gems ridley and ffi-libarchive.
+* Disable libvirtd service on CentOS 7. Virtual bridge interfaces are incorrectly detected by Open MPI and
+  cause MPI applications to hang, see https://www.open-mpi.org/faq/?category=tcp#tcp-selection for details 
+* Use CINC instead of Chef for provisioning instances. See https://cinc.sh/about/ for details.
+* Retry when mounting an NFS mount fails.
+* Install the `pyenv` virtual environments used by ParallelCluster cookbook and node daemon code under
+  /opt/parallelcluster instead of under /usr/local.
+* Avoid downloading the source for `env2` at installation time.
```
Comment (mauri-melato) [inline CHANGELOG.rst:36]: Is it really worth to be mentioned? we don't need to report here all the details of -node and -cookbook projects

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,60 @@
...
+* Vendor cookbooks as part of instance provisioning, rather than doing so before copying the cookbook into an
```
Comment (mauri-melato) [inline CHANGELOG.rst:37]: Let's start from the impact to customers, in a form that's comprehensible so related to the functionality of the product and not to technical insights: Remove the client requirement to have berks (or ChefDK) to build a custom AMI.
This is worth to be mentioned at the top of the list for its impact.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,60 @@
...
+  instance. Users no longer need to have `berks` installed locally.
+* Drop the dependencies on the poise-python, tar and hostname third-party cookbooks.
```
Comment (mauri-melato) [inline CHANGELOG.rst:39]: Not worht to be mentioned here

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,60 @@
...
+* Use the new official CentOS 7 AMI as the base images for ParallelCluster AMI.
+* Upgrade NVIDIA driver to Tesla version 440.95.01 on CentOS 6 and version 450.51.05 on all other distros.
+* Upgrade CUDA library to version 11.0 on all distros besides CentOS 6.
+* Install third-party cookbook dependencies via local source, rather than using the Chef supermarket.
+* Use https wherever possible in download URLs.
+* Upgrade EFA installer to version 1.9.4:
```
Comment (mauri-melato) [inline CHANGELOG.rst:45]: One at the top of this list.

File: CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,60 @@
...
+  * Kernel module: ``efa-1.6.0`` (from efa-1.5.1)
+  * RDMA core: ``rdma-core-28.amzn0`` (from rdma-core-25.0)
+  * Libfabric: ``libfabric-1.10.1amazon1.1`` (updated from libfabric-aws-1.9.0amzn1.1) 
+  * Open MPI: openmpi40-aws-4.0.3 (no change)
+
+**BUG FIXES**
+
+* Fix validator for CIDR-formatted IP range parameters.
+* Fix function used to convert the `extra_json` parameter to/from CloudFormation parameters.
```
Comment (mauri-melato) [inline CHANGELOG.rst:54]: Useful to know the issue we fixed and then where we fixed it

File: CHANGELOG.rst [hunk 7]
```diff
@@ -2,6 +2,60 @@
...
+* Fix issue that was preventing concurrent use of custom node and pcluster CLI packages.
+* Use the correct domain name when contacting AWS services from the China partition.
+* Avoid pinning to a specific release of the Intel HPC platform.
```
Comment (mauri-melato) [inline CHANGELOG.rst:57]: Let's be more precise here, we *are pinning* major minor version but not the precise patch release number.

File: CHANGELOG.rst [hunk 8]
```diff
@@ -2,6 +2,60 @@
...
+2.8.0
+=====
+
+**ENHANCEMENTS**
+
+* Print a warning when using NICE DCV on micro or nano instances.
+* Add support for S3 URI in custom_chef_cookbook.
+* Make `pcluster update` command safe.
+* Support DCV in China partition.
+* Use describe_instance_types to validate whether an instance type is EFA-enabled.
```
Comment (mauri-melato) [inline CHANGELOG.rst:14]: How is this going to impact the user, either reword or remove it.

File: CHANGELOG.rst (hunk 7 above)
Comment (mauri-melato) [reply CHANGELOG.rst:57]: Overall, it's not even worth to mention here.

File: CHANGELOG.rst [hunk 9]
```diff
@@ -2,6 +2,60 @@
...
+* Enable support for ARM instances on Ubuntu 18.04 and Amazon Linux 2.
+* Support EFS in the China partition.
```
[the remark this answers is not in the document]
Comment (tilne) [reply CHANGELOG.rst:16]: I combined this and the DCV message into the same item, 4th in the list

File: CHANGELOG.rst (hunk 1 above)
Comment (tilne) [reply CHANGELOG.rst:27]: It's to avoid extra API calls for IAM policies we already know are valid. I'll reword it.

"Avoid validation of IAM policies known to be valid."

File: CHANGELOG.rst (hunk 6 above)
Comment (tilne) [reply CHANGELOG.rst:55]: I believe this identifies the what. What do you mean by where we fixed it?

File: CHANGELOG.rst [hunk 10]
```diff
@@ -2,6 +2,60 @@
...
+* Enable configuration of the automatic backup features of FSx file systems.
+* Install  PMIx v3.1.5 and provide slurm support for it on all supported operating systems except for
```
[the remark this answers is not in the document]
Comment (tilne) [reply CHANGELOG.rst:18]: "Enable Slurm to directly launch tasks and initialize communications through PMIx v3.1.5 on all supported operating systems except for CentOS 6."

File: CHANGELOG.rst [hunk 11]
```diff
@@ -2,6 +2,54 @@
...
+* Support DCV and EFS in China regions.
+* Use DescribeInstanceTypes API to validate whether an instance type is EFA-enabled so that new EFA instances can
+  be used without requiring an update to the ParallelCluster configuration files.
+* Enable Slurm to directly launch tasks and initialize communications through PMIx v3.1.5 on all supported
+  operating systems except for CentOS 6.
+* Print a warning when using NICE DCV on micro or nano instances.
+
+**CHANGES**
+
+* Remove the client requirement to have berks (or ChefDK) to build a custom AMI.
```
Comment (mauri-melato) [inline CHANGELOG.rst:22]: berks -> Berkshelf  
please remove the "(or ChefDK)" that we're not mentioning it anymore as dependency in our doc here  https://docs.aws.amazon.com/parallelcluster/latest/ug/pcluster.createami.html

File: CHANGELOG.rst [hunk 12]
```diff
@@ -2,6 +2,54 @@
...
+* Upgrade EFA installer to version 1.9.4:
+  * Kernel module: ``efa-1.6.0`` (from efa-1.5.1)
+  * RDMA core: ``rdma-core-28.amzn0`` (from rdma-core-25.0)
+  * Libfabric: ``libfabric-1.10.1amazon1.1`` (updated from libfabric-aws-1.9.0amzn1.1) 
+  * Open MPI: openmpi40-aws-4.0.3 (no change)
+* Migrate S3 URLs to Virtual Hosted Style Access.
```
[the remark this answers is not in the document]
Comment (tilne) [reply CHANGELOG.rst:28]: i'll remove it
